sycl/doc/extensions/experimental/sycl_ext_intel_data_flow_pipes_properties.asciidoc

```diff
@@ -178,10 +187,13 @@ The number of cycles between when the ready signal is deasserted and when the
 pipe can no longer accept new inputs.
 
 This property is not guaranteed to be respected if the pipe is an inter-kernel
-pipe. The compiler is allowed to optimize the pipe if both sides are visible.
+pipe. 
+
+This property will only have an effect when the `protocol` property is *avalon_streaming*
```

Comment:
I think we probably want to be more stringent and say that it's _only valid_ on avalon streaming pipes.  Otherwise it's easy for users to make mistakes.